lab_3_ann_retriever/main.py

```diff
@@ -5,61 +5,47 @@
 """
 
 # pylint: disable=too-few-public-methods, too-many-arguments, duplicate-code, unused-argument
+from lab_2_retrieval_w_bm25.main import (calculate_idf, calculate_tf)
+from re import sub
 from typing import Protocol
 
-Vector = tuple[float, ...]
-"Type alias for vector representation of a text."
-
 
 class NodeLike(Protocol):
-    """
-    Type alias for a tree node.
-    """
+    """Type alias for a tree node."""
 
     def save(self) -> dict:
         """
         Save Node instance to state.
 
         Returns:
-            dict: State of the Node instance
+            dict: state of the Node instance
```

Comment:
revert

Reply:
i have been working on this lab since friday 1/11. on saturday some things in this lab - including docstrings, as ive just discovered - were changed and i had to use `git pull` so that i could push my lab to open a new pull request. then i lost all my progress and had to revert _external change_. i didnt notice docstrings being changed, as the changes werent that monumental. i guess that is why it seems like **i** changed them... even though i did not
moreover, i feel offended by your claim about my usage of AI as i dont use it. if you dont believe me, you can check my local history any time...

Reply:
dont worry. These changes are seemed really strange, thats why I began not to understand their origin. I believe in you, of course.

Reply:
so can you revert all these changes somelike? If you have some questions please write in chat

Reply:
i have some probable solution in my head and i will implement it sometime before the last mentor check (sorry) as i got too attached to the old version of the lab

Reply:
so, i am waiting :)